manifests/sentinel.pp

```diff
@@ -129,13 +157,22 @@
   Integer[0] $parallel_sync = 1,
   Stdlib::Absolutepath $pid_file = $redis::params::sentinel_pid_file,
   Integer[1] $quorum = 2,
+  Optional[Boolean] $sentinel_announce_hostnames = undef,
```

Comment:
This and `sentinel announce-hostnames` should be an `Enum['yes', 'no']` instead of Boolean that converts to yes or no.

Optional can remain I think

Reply:
What is the standard you are using?  The rest of the code seems to use Boolean when it's yes, no unless there is another possible value.

Reply:
For example: 
protected-mode <%= @protected_mode ? 'yes' : 'no' %>

Reply:
We try to use Enum instead of booleans convering to some values

https://voxpupuli.org/docs/reviewing_pr/

> Does a new param map to an option in a config file of a service? The Parameter should accept the possible values that the service allows. For example ‘on’ and ‘off’. Don’t accept a boolean that will be converted to ‘on’ or ‘off’

Reply:
okay.  I will fix.  I was just trying to keep things consistent with what was already there.